runtime/criusupport/criusupport.cpp

```diff
@@ -516,9 +516,7 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		Trc_CRIU_checkpoint_nano_times(currentThread, checkpointNanoTimeMonotonic, checkpointNanoUTCTime);
 		TRIGGER_J9HOOK_VM_PREPARING_FOR_CHECKPOINT(vm->hookInterface, currentThread);
 
-		/* trigger a GC to disclaim memory */
-		vm->memoryManagerFunctions->j9gc_modron_global_collect_with_overrides(currentThread, J9MMCONSTANT_EXPLICIT_GC_SYSTEM_GC);
-		vm->memoryManagerFunctions->j9gc_modron_global_collect_with_overrides(currentThread, J9MMCONSTANT_EXPLICIT_GC_PREPARE_FOR_CHECKPOINT);
+		vm->memoryManagerFunctions->j9gc_criu_checkPoint(currentThread);
```

Comment:
I'd try to not use criu specific names. in future, on other OSes some other underlying technology might be available, but the high-level logic around snapshot (checkpoint) and restore is same.

short like:
j9gc_pre_checkpoint/post_restore
or longer like:
j9gc_prepare_for_checkpoint/adjust(reinitialize)_after_restore